practice_project/isPrime.py

```diff
@@ -0,0 +1,17 @@
+def is_prime(number) :
```

Comment:
I would recommend adding type hints for each method parameter and the return type of the method so that your IDE highlights misuse of the method (i.e. when you pass in a wrong type or you expect a wrong type of what is being returned